bin/update_dev_data.py

```diff
@@ -86,11 +87,20 @@ def upsert_application_instance(self, data):
         self.setattrs(application_instance, data)
 
     def upsert_assignment(self, data):
-        assignment = (
-            self.db.query(models.Assignment)
-            .filter_by(resource_link_id=data["resource_link_id"])
-            .one_or_none()
-        )
+        try:
+            assignment = (
+                self.db.query(models.Assignment)
+                .filter_by(
+                    resource_link_id=data["resource_link_id"],
+                    tool_consumer_instance_guid=data["tool_consumer_instance_guid"]
+                )
+                .one_or_none()
+            )
+
+        except MultipleResultsFound:
+            print(f"Found more than one row for {data['resource_link_id']} in the DB")
+            print(f"We were going to upsert: {data}")
+            raise
```

Comment:
I think we can remove the `except`. There's a [unique constraint on `(resource_link_id, tool_consumer_instance_guid)`](https://github.com/hypothesis/lms/blob/d62445c2c020eb54c78060c33a943d07938f8a30/lms/models/assignment.py#L26) so the code is gonna work correctly now. Bug fixed